src/components/Features/Transfer/Transfer.js

```diff
@@ -202,27 +164,31 @@ export const Transfer = () => {
   };
 
   return (
-    <Menu>
-      <div className={styles.transfer}>
-        <div className={styles.tabsContainer}>{renderTabs()}</div>
-        {!selectedToken && (
-          <center>
-            <Loading size={LoadingSize.XL} />
-          </center>
-        )}
-        {selectedToken && (
-          <>
-            {isL1 ? renderL1Network() : renderL2Network()}
-            <NetworkSwap isFlipped={isL2} onClick={onSwapClick} />
-            {isL1 ? renderL2Network() : renderL1Network()}
-          </>
-        )}
-        {isLoggedIn ? (
-          <TransferButton isDisabled={isButtonDisabled || bridgeIsFull} onClick={onTransferClick} />
-        ) : (
-          <LoginWalletButton />
-        )}
-      </div>
-    </Menu>
+    <>
+      {!selectedToken && (
```

Comment:
`selectedToken ? ... : ...` will make the same work

Reply:
@galuzan-starkware it's not relevant to my change (hide whitespace to see it easier)